.github/workflows/ci-uto.yaml

```diff
@@ -0,0 +1,189 @@
+name: Build and test Universal Transfer Operator
+on:
+  push:
+    branches: [ 'main', 'release-**' ]
+    paths:
+      - 'python-sdk/**'
+      - 'universal_transfer_operator/**'
+
+  pull_request:
+    branches: [ 'main', 'release-**' ]
+    paths:
+      - 'python-sdk/**'
```

Comment:
Why do we want to trigger the UTO tests when there is a change to the `python-sdk`?